source/_integrations/august.markdown

```diff
@@ -67,8 +68,6 @@ Other August locks expect to be powered by AA alkaline (non-rechargeable) batter
 
 ## Known Unsupported Devices
 
-- The Yale Doorman L3
-
 Other devices not listed above have not been tested and may not function as expected.
```

Comment:
As there is no longer a list present, this looks strange now.

Reply:
As there is no known unsupported, can we just remove this section?

Reply:
Yes, but the unintended consequence will be users will ask me why a device doesn't work and I'll have nothing to point them to

Reply:
ea05e4dd31d50547c9bd858e53a09491ab0b3e95